ghost/members-api/lib/repositories/MemberRepository.js

```diff
@@ -281,7 +282,29 @@ module.exports = class MemberRepository {
             memberStatusData.status = 'comped';
         }
 
-        // Subscribe member to default newsletters
+        //checks for custom signUp forms
+        if (memberData.type === 'subscribe' && memberData.newsletters) {
+            const allValidNewsletters = await this._newslettersService.browse({limit: 'all'});
```

Comment:
Instead of querying for all newsletters, could we just query for the specific newsletter we are looking for with a `filter` param? i.e:

```js
options.filter = 'status:active+id:<ID_OF_NEWSLETTER>'

await this._newslettersService.browse(options);

Reply:
aaah...I was exploring this options.filter yesterday. But couldn't find a way to do IN query. 
Is there a way to make an IN query using options.filter? 
Basically this: `SELECT * FROM newsletters WHERE id IN ('asdf3456788dfgvh', 'fdghv4567dftgy');` 

Because otherwise if I make a query using 'status:active+id:<ID_OF_NEWSLETTER>' I'll have to do a separate query for all the subscribed newsletters and that's too many unnecessary db calls when we can get the data in one call.

Reply:
> I'll have to do a separate query for all the subscribed newsletters 

Remember for the 🛹 we are only interested in a single newsletter

Reply:
Simplified this for 🛹